### PR TITLE
Remove errors caused by explicit `.colorize.rst` include

### PR DESCRIPTION
**Pull request by jd41**
Update: I got this PR wrong the first time. I wrote an update and hope it's correct now! I also put my current understanding into comments in `colorize.rst`.

I
- removed the explicit includes of `.colorize.rst` in `conf.py` and `examples/index.rst`. If I checked correctly, Sphinx recognizes the roles introduced therein throughout the toctree without any explicit `include`. The `include colorize.rst` in the `conf.py` `epilog` variable was missing a '/' before `.colorize.rst`, I think. Because of that, Sphinx always looked for `.colorize.rst` in the same folder as the file it currently rendered. This only worked in the doc root folder and caused ~300 errors during build everywhere else.
- added `:orphan:` role to `.colorize.rst` to prevent not-included-in-toctree warning
- renamed `.colorize.rst` to `colorize.rst`. The leading dot seemed to have no effect to me except on UNIX ls, in particular, `.colorize.rst` was still being turned into an orphan [HTML page](https://nest-simulator.readthedocs.io/en/latest/.colorize.html). The dot confused me for some time as I mistakenly thought it had a special meaning.
- wrote the understanding I got about how this works in comments in `colorize.rst`

### CODE REVIEW

**Comment by jd41** [comment]
Before anyone looks at this: Sorry, I hit the submit button too soon on this one. I am currently working on a fix and will then update the first message.

**Comment by jd41** [comment]
Updated now with new commits. I hope it works out now!

**Review by steffengraber — APPROVED** [review]
Tested and it works. Thanks Joris for this correction.

File: doc/colorize.rst [hunk 1]
```diff
@@ -1,5 +1,10 @@
+:orphan:
 .. Color profiles for Sphinx.
 .. Has to be used with hacks.css (bitbucket.org/lbesson/web-sphinx/src/master/.static/hacks.css)
+.. In the NEST Doc, the content of hacks.css is included in _static/css/custom.css
+.. Even without explicitly including this file, the color roles herein can be used throughout the documentation
+.. unfortunately, it still gets rendered as a HTML file, but it is not included in any toctree
```
Comment (sarakonradi) [inline doc/colorize.rst:6]:
```suggestion
.. It gets rendered as an HTML file but is not included in any toctree
```

File: doc/colorize.rst [hunk 2]
```diff
@@ -1,5 +1,10 @@
+:orphan:
 .. Color profiles for Sphinx.
 .. Has to be used with hacks.css (bitbucket.org/lbesson/web-sphinx/src/master/.static/hacks.css)
+.. In the NEST Doc, the content of hacks.css is included in _static/css/custom.css
```
Comment (sarakonradi) [inline doc/colorize.rst:4]:
```suggestion
.. In the NEST documentation, the content of hacks.css is included in _static/css/custom.css
```

**Review by sarakonradi — CHANGES_REQUESTED** [review]
Thanks! It works. Please find my comments inline.

**Comment by heplesser** [comment]
@jd41 With #1839 and #1852, we fixed several problems in the Travis setup. Please merge master into the branches in your PRs to ensure that testing will work properly on all systems.

**Comment by jessica-mitchell** [comment]
Closing and will be addressed by @jessica-mitchell

**Pull request closed without merging**